setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name="fastapi_versioning",
-    version="0.3.0",
+    version="0.3.1",
```

Comment:
This change isn't needed, I use `bumpversion` to manage this version number.

Reply:
Great, I'll give a try to `bumpversion` as well :)